integrations/management/commands/health_check.py

```diff
@@ -0,0 +1,139 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from decouple import config
+from hubspot import HubSpot
+from hubspot.crm.contacts.exceptions import ForbiddenException
+from django.db.models import Q
+from authentication.models import User
+from integrations.models import Link
+from programs.models import Navigator, Program, TranslationOverride, UrgentNeed
+from translations.models import BLANK_TRANSLATION_PLACEHOLDER, Translation
+from configuration.models import Configuration
+import argparse
+import json
+
+
+class Command(BaseCommand):
+    help = "Check that we can't read from HubSpot, and there is no PII in the database"
+
+    HUB_SPOT_TEXT = "Can't read Hub Spot"
+    PII_IN_DB_TEXT = "PII not in the database"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-s",
+            "--strict",
+            action=argparse.BooleanOptionalAction,
+            help="Should the he website hashes should be compared to the old hashes",
```

Comment:
Is this supposed to say "The website hashes should be compared to the old hashes"?